### PR TITLE
Added About/Help Page

### PR DESCRIPTION
**Pull request by phuang00**
- added tooltips for confusing components
- renamed History to Deleted Items
- populated About/Help page with the motivation of the app as well as how to use it (TODO: will need to populate with recipe synchronizations once merged)

### CODE REVIEW

File: client/components/Basket/BasketsPage.vue [hunk 1]
```diff
@@ -4,7 +4,8 @@
   <main>
     <AddBasketForm />
     <hr>
-    <h2 class="display-4">My Baskets</h2>
+    <h1>My Baskets</h1>
+    <p>Keep track of all the different buckets of items you want to buy here.</p>
```
Comment (jlin00) [inline client/components/Basket/BasketsPage.vue]: Change to "Keep track of different groups of food items here."

File: client/components/DeletedItem/DeletedItemsPage.vue [hunk 2]
```diff
@@ -2,7 +2,8 @@
 
 <template>
   <main>
-    <h2 class="display-4">History</h2>
+    <h1>Deleted Items</h1>
+    <p>View all your previously deleted pantry items here.</p>
```
Comment (jlin00) [inline client/components/DeletedItem/DeletedItemsPage.vue]: Remove the word previously.

File: client/components/common/HelpPage.vue [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+          <img src="@/public/basketsPage.png" style="max-width: 80%"/>
+          <p>When creating baskets, you first need to name it and optionally, populate it with items, one at a time:</p>
+          <img src="@/public/createBasket.png" style="max-width: 80%"/>
+          <p>Once on an individual baskets page, you can modify/delete the basket. When you've bought items in a basket, you can also express add it to the pantry:</p>
+          <img src="@/public/basketPage.png" style="max-width: 80%"/>
+          <p>Next up, we have recipes, a place for you to explore recipes:</p>
+          <img src="@/public/recipesPage.png" style="max-width: 80%"/>
+          <p>You can upload your own recipes:</p>
+          <img src="@/public/addRecipe.png" style="max-width: 80%"/>
+          <p>Or filter through already uploaded recipes for ones that contain both the inputed keyword and ingredients: </p>
```
Comment (jlin00) [inline client/components/common/HelpPage.vue]: typo in the word inputted

File: client/components/common/NavBar.vue [hunk 4]
```diff
@@ -61,6 +61,15 @@
           <span class="fs-4">Recipes</span>
         </router-link>
       </li>
+      <li class="nav-item mb-3">
+        <router-link 
+          class="text-white"
+          to="/help"
+        >
+        <i class="bi bi-info-square mx-2"></i>
```
Comment (jlin00) [inline client/components/common/NavBar.vue]: I think info-circle looks better.

**Review by jlin00 — CHANGES_REQUESTED** [review]
Everything looks good, minor changes requested.

**Comment by phuang00** [comment]
updated

**Review by jlin00 — APPROVED** [review]

**Pull request merged**